src/time_integration/paired_explicit_runge_kutta/methods_PERK3.jl

```diff
@@ -0,0 +1,409 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+using DelimitedFiles: readdlm
+
+@muladd begin
+#! format: noindent
+
+# Initialize Butcher array abscissae c for PairedExplicitRK3 based on SSPRK33 base method
+function compute_c_coeffs_SSP33(num_stages, cS2)
+    c = zeros(num_stages)
+
+    # Last timesteps as for SSPRK33
+    c[num_stages] = 0.5
+    c[num_stages - 1] = 1
```

Comment:
We should change the order here
```suggestion
    c[num_stages - 1] = 1
    c[num_stages] = 0.5
```